.ci/setup.sh

```diff
@@ -38,6 +38,8 @@ if [ -n "$PULL_REQUEST_NUMBER" ] || [ -n "$LOCALCI_PR_NUMBER" ]; then
 	bash -f "${cidir}/run_fetch_branches_tool.sh"
 fi
 
+echo "Install go"
+bash -c "${cidir}/install_go.sh -p"
```

Comment:
@grahamwhaley  @jodh-intel @chavafg @sboeuf  what do you think about this, do you think we can call this script in the Jenkins setup instead?

Reply:
I think we can, as long as we define when and where well.
If we were to do that - I'd like to see a check that *only* installs if the version is different from what is already installed - no point installing the same go version over and over again (on a bare metal instance) iyswim